src/Microsoft.CodeQuality.Analyzers/Core/Performance/MicrosoftPerformanceAnalyzersResources.resx

```diff
@@ -0,0 +1,129 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Can we avoid creating new resource files and folder(s) for this rule here? You can move the rule under "QualityGuidelines" folder and use the existing resx file. For example, see this performance rule under it: https://github.com/dotnet/roslyn-analyzers/blob/ac2cae9dd4807007e7ae0d25884a3d734703636d/src/Microsoft.CodeQuality.Analyzers/Core/QualityGuidelines/PreferJaggedArraysOverMultidimensional.cs#L32

FYI: Multiple resx files inside a single project is an unfortunate consequence of us previously having these rules spread across different projects such as Maintenance, QualityGuidelines, etc. That was later refactored to have all of them in a  single Microsoft.CodeQuality.Analyzers project, and we just moved the sources and resx files as is. We haven't yet got to merging all the resx files into a single one, but we should certainly avoid creating more resx files inside each project.

Reply:
@mavasani,

I followed what I perceived to be the standard for the projects.

Are saying to move the analyzer from the `Performance` folder to the `QualityGuidelines` folder or to use the resources in the `QualityGuidelines` folder? I don't think either one makes sense because it would break the relation between the namespace and the folder name.

Wouldn't it make more sense to have a top level resource file for each project and move all resources there?

Reply:
Sorry, I meant what you suggested in the last statement is _exactly_ what we should do - have a single resource file per project. We have multiple of them because of historical reasons, and I'd prefer if we not add one more to the project.

Reply:
Moved just the resources for this analyzer.
I'll come up with something to batch change all others.